### ISSUE TITLE
[BUG] "EC2Client" has no attribute "meta"

### ISSUE THREAD

**Issue by tekumara** [vemel/mypy_boto3_builder#45]
**Describe the bug**

The EC2Client stubs are missing the attribute `meta`.

**To Reproduce**
1. Install 'boto3-stubs[ec2]'
2.  Run mypy on the following code sample

```python
import boto3 

ec2_client = boto3.client("ec2")

print(ec2_client.meta.region_name)
``` 
3. Actual mypy output

```
test.py:5: error: "EC2Client" has no attribute "meta"
Found 1 error in 1 file (checked 1 source file)
```

**Expected mypy-output**

No type error.

**Additional context**

boto3-stubs==1.14.46.0

**Comment by tekumara** [comment]
`meta` looks undocumented, as I don't see it mentioned in the [boto 3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#client).

**Comment by tekumara** [comment]
I've found mention of `meta` in the [extensibility guide](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/events.html) so it is (somewhat) documented.

**Comment by vemel** [comment]
Thank you for the report. Yes, each client has a `meta`. I will try to add it to stubs.

**Comment by vemel** [comment]
Hello!

This was fixed in boto3_stubs>=1.14.51.1. Let me know if the issue still exists.
